pkg/sources/github/github.go

```diff
@@ -702,8 +701,8 @@ func (s *Source) enumerateWithApp(ctx context.Context, apiEndpoint string, app *
 			if err != nil {
 				return nil, err
 			}
-			s.log.Info("Scanning repos", "org_members", len(s.members))
-			for _, member := range s.members {
+			s.log.Info("Scanning repos", "org_members", len(s.memberCache))
+			for member, _ := range s.memberCache {
```

Comment:
nit: looks like an unnecessary assignment here since we just need the member key.

Reply:
Good point. I'll remove that.